### ISSUE TITLE
how can i  getNinePatchChunk()

### ISSUE THREAD

**Issue by wlssing** [facebook/fresco#1604]
this bitmap is a NinePatchDrawable
i can get data from bitmap (bitmap .getNinePatchChunk()) when it loaded from picasso 
but I get a bitmap from network by using fresco,i cannot get null

how can i get data(bitmap .getNinePatchChunk())

**Comment by lambdapioneer** [comment]
Hi @ccwls3290, can you provide a code sample that reproduces the issue? And can you rephrase what you are trying to achieve?

**Comment by ccwls3290** [comment]
private Drawable getNinePatchDrawable(Bitmap bitmap,Context context) {
	    
        byte[] chunk =bitmap.getNinePatchChunk();
        NinePatchDrawable ninePatchDrawable = null;
        if(NinePatch.isNinePatchChunk(chunk))
        {
            ninePatchDrawable = new NinePatchDrawable(context.getResources(), bitmap, chunk,
                    new Rect(), null);
        }
        return ninePatchDrawable;
    }

the bitmap from Picasso run but the bitmap from fresco

**Comment by erikandre** [comment]
Closing as this is enhancement is not prioritized to be implemented.
Still, please feel free to submit a pull request for any missing functionality.
